bin/main

```diff
@@ -1,72 +1,55 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-turn = 0
-player = true
-choices = [0]
+require_relative '../lib/board'
+require_relative '../lib/selection_display'
 
-# Check for win and display winner
-def winner
-  puts 'Here we define if a winner is found if it is we display the winner'
-  false
-end
-# Here we end the game either called by winner method or if the turn is > 9
-
-def game_over
-  puts 'Game over'
-end
-puts 'The game is on!'
-puts '-----------------------'
-puts 'Game will start select a valid input for your play (Any number from 0~9)'
+board_controller = Board.new
+local_selector = Selector.new
 
-# Displaying the board
-puts 'Display the game board'
 
-while turn < 9
-  if player
-    puts "Player One's turn"
-    player_one = gets.to_i
-    if player_one < 1 || player_one > 9
-      puts 'Wrong input'
-    else
-      puts "Player's selection displayed one the board"
-      player = false
-      if choices.include? player_one
-        puts 'Choice already taken, Please choose another choice!'
-      else
-        choices << player_one
-        turn += 1
-
-        win = winner
-        if win
-          # Display the player who has won
-          game_over
-        end
-      end
+def init_board(board_c)
+  puts '--------------'
+  board_c.board.each do |n|
+    puts n.join '  |  '
+    puts '--------------'
+  end
+end
```

Comment:
**OPTIONAL**
move that method into the **Board** class
```ruby
def init_board
   result = ''
   @board.each do |n|
        result += n.join '   |   '
        result += '\n--------------\n'
    end
end
```

Then to print the board you just call it like this
```ruby
puts board_c.init_board
```

Reply:
Good idea
Fixed